includes/rest/class-actors-controller.php

```diff
@@ -42,10 +42,11 @@ public function register_routes() {
 			array(
 				'args'   => array(
 					'user_id' => array(
-						'description' => 'The ID or username of the actor.',
-						'type'        => 'string',
-						'required'    => true,
-						'pattern'     => '[\w\-\.]+',
+						'description'       => 'The ID of the actor.',
+						'type'              => 'string',
+						'required'          => true,
+						'pattern'           => '-?\d+',
```

Comment:
Should the type be changed to `integer`? Not sure if that works with `-1`.

I don't know why I added the pattern here initially, it shouldn't matter. If it doesn't match the rest_base pattern we never even get here.